apache/server/service/init.sls

```diff
@@ -57,6 +59,14 @@ apache_service:
   - require:
     - pkg: apache_packages
 
+{% else %}
+
+apache_service:
+  service.dead:
```

Comment:
noservices means - do not touch services, so it shouldn't even ensure they are dead.